NetworkingTests/Tests/SessionManagerSpec.swift

```diff
@@ -288,11 +288,6 @@ internal class SessionManagerSpec: QuickSpec {
                     sessionManager.bootstrap()
                 }
                 
-                it("sends no session signal") {
```

Comment:
This test should be kept, why did you remove it?

Reply:
I meant the other test but he told me he talked this one with you too